packages/ui-kit/src/components/TimeSeries/TimeSeries.tsx

```diff
@@ -148,9 +148,13 @@ export const TimeSeriesComponent: React.FC<TimeSeriesProps> = ({
       if (!canvasRef.current || !data?.labels || !data.values || hasError || (variant !== 'bar' && variant !== 'line'))
         return
 
-      const labels = data.labels || []
+      let labels = data.labels || []
       const values = data.values || []
 
+      if (labelFormatter) {
+        labels = formatLabels({ labels, formatter: labelFormatter })
+      }
+
```

Comment:
I think it would be nice if we keep `labels` as `const`, you can even do something like this because `formatLabels` will return the original labels in case the formatter is `null`
```suggestion
      const labels = formatLabels({ labels: data.labels, formatter: labelFormatter }) || []
      const values = data.values || []
```

Reply:
Thanks! Done!

However, hiding the checker for the `labelFormatter` existence in `formatLabels` method makes code less readable versus having it on the top level, e.g.
```tsx
// Okay, here are my labels
let labels = data.labels || []

// Okay, in case I don't pass the `labelFormatter` prop the labels stay the same,
// otherwise some additional logic will apply, and I'll have to look `labelFormatter`
// code for more details.
if (labelFormatter) {
  labels = formatLabels({ labels, formatter: labelFormatter })
}
```
vs.
```tsx
// Okay, there is some formatting happening in `formatLabels` method
// and I have to check its code anyway.
const labels = formatLabels({ labels: data.labels, formatter: labelFormatter }) || []
```

Not a big deal though. Eventually, we may also extend the logic of `formatLabels` method.